ui.apps/src/main/content/jcr_root/apps/core/fd/components/form/text/v1/text/_cq_editConfig.xml

```diff
@@ -0,0 +1,180 @@
+                xtype="richtext">
+            <rtePlugins jcr:primaryType="nt:unstructured">
+                <footnoteplugin
+                        jcr:primaryType="nt:unstructured"
+                        features="*"/>
+                <links jcr:primaryType="nt:unstructured">
+                    <linkDialogConfig
+                            jcr:primaryType="nt:unstructured"
+                            height="{Long}316">
+                        <linkAttributes jcr:primaryType="cq:WidgetCollection">
+                            <linkAdvanced
+                                    jcr:primaryType="cq:Widget"
+                                    collapsed="{Boolean}true"
+                                    collapsible="{Boolean}true"
+                                    inputValue="advanced"
+                                    name="./linkdialog/cq:adhocLinkTrackingTab"
+                                    title="Link tracking"
+                                    xtype="dialogfieldset">
+                                <items jcr:primaryType="cq:WidgetCollection">
+                                    <enable
+                                            jcr:primaryType="nt:unstructured"
+                                            attribute="enabletracking"
+                                            fieldDescription="override analytics framework settings."
+                                            fieldLabel="Custom link tracking"
+                                            name="./linkdialog/cq:adhocLinkTrackingEnableTracking"
+                                            xtype="checkbox">
+                                        <listeners
+                                                jcr:primaryType="nt:unstructured"
+                                                check="function(component){var dlg=component.findParentByType('rtelinkdialog');dlg.enableSCFields(component.checked);}"/>
+                                    </enable>
+                                    <events
+                                            jcr:primaryType="nt:unstructured"
+                                            attribute="adhocevents"
+                                            fieldDescription="e.g.: event2, event7."
+                                            fieldLabel="Include SiteCatalyst events"
+                                            name="./linkdialog/cq:adhocLinkTrackingEvents"
+                                            xtype="textfield"/>
+                                    <evars
+                                            jcr:primaryType="nt:unstructured"
+                                            attribute="adhocevars"
+                                            fieldDescription="e.g.: eVar1: pagedata.url, prop4: 'const'."
+                                            fieldLabel="Include SiteCatalyst variables"
+                                            name="./linkdialog/cq:adhocLinkTrackingEvars"
```

Comment:
what are these ?

Reply:
They were coming from editconfig of foundation component. Commented out since they are not available in rte plugin anymore.